spec/models/journal_entry_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
```

Comment:
add validations for `dependent_destroy` and `inverse_of` as well (wherever applicable)